### ISSUE TITLE
403 and JSON.stringify()ing error

### ISSUE THREAD

**Issue by SpellChucker** [getsentry/sentry-cordova#18]
I followed all instructions for our Ionic app [here](https://docs.sentry.io/clients/cordova/ionic/), and it works perfectly on web. However, whenever I build it (at least for iOS), I get the following errors:

`Sentry - Error:: Request failed: Error Domain=SentryErrorDomain Code=106 "Request errored with 403" UserInfo={NSLocalizedDescription=Request errored with 403}`

and

`ERROR: error JSON.stringify()ing argument: TypeError: JSON.stringify cannot serialize cyclic structures.`

Any ideas where these are coming from? I'm building with the usual `ionic cordova build ios`

**Comment by HazAT** [comment]
Hey, how does your code look like, where you use Sentry?
It seems that you somewhere set a cyclic object into the context.

**Comment by SpellChucker** [comment]
I set it up just like the sentry instructions have:

```
 Sentry.create(configService.get('sentryUrl'))
   .use(SentryCordova, {sentryBrowser: SentryBrowser})
   .install();

 export class SentryIonicErrorHandler extends IonicErrorHandler {
   public handleError(error) {
     super.handleError(error);
     try {
       Sentry.getSharedClient().captureException(error.originalError || error);
     } catch (e) {
       console.error(e);
     }
   }
 }
```
We have a `ConfigService` that provides us with the correct DSN to use for Sentry.

Also have the `{ provide: ErrorHandler, useClass: SentryIonicErrorHandler }` setup.

Before we were using the Raven approach but this plugin is much much better (when it works :))

**Comment by SpellChucker** [comment]
So weird thing, I changed our sentry DSN to use the complete DSN (the non-public version) and it's working great now! Very weird errors for that though.

**Comment by HazAT** [comment]
OK, thank you for reporting this.
I will still check what actually happened here.

**Comment by HazAT** [comment]
This should be resolved in the latest version `0.8.0`
